vendor/assets/stylesheets/jquery-bar-rating.css

```diff
@@ -0,0 +1,4 @@
+/*
+  *= require_tree .
```

Comment:
I'd move all themes into a dedicated directory and require that directory instead